README.md

```diff
@@ -214,7 +216,7 @@ Please note that setting a custom endpoint will stop Crash Report or Real User M
 
 ### Storing crash reports on the device
 
-If the device can't connect, Raygun4Android will save the crash report to disk. At the next start of the application (and therefore the provider) it will check if the internet is now available, and if it is, send the cached messages. A maximum of 64 messages will be cached and you can change the amount by calling:
+If the device can't connect, Raygun4Android will save the crash report to storage. At the next start of the application (and therefore the provider) it will check if the internet is now available, and if it is, send the cached messages. A maximum of 64 messages will be cached and you can change the amount by calling:
```

Comment:
If the storage limit (with the default of 64) is reached, it will just not write more unless there is space. If that's a good way of doing it, I'm not certain - but it was at the time aligning how Mitchell had written the iOS provider, so we tried to keep the behaviour in line.

I think the default of 64 is not really that reasonable anymore these days either - we should consider to change that maybe. There's also https://github.com/MindscapeHQ/raygun4android/issues/49 - that goes into that direction of "we should really consider changing that" as well.

Reply:
Just pushed an updated with a note on the current behaviour.